### PR TITLE
Stop UI server from using stdout/stderr inadvertently

### PR DESCRIPTION
**Pull request by cretz**
## Description & motivation 💭

Stop UI server from using stdout/stderr inadvertently

### CODE REVIEW

**Review by lorensr — APPROVED** [review]
Chad and I tested CLI server with this change

**Comment by cretz** [comment]
Will wait for approval from frontend before merging...

**Review by Alex-Tideman — APPROVED** [review]

**Pull request merged**
